engine/src/main/scala/cromwell/engine/backend/jes/JesConfiguration.scala

```diff
@@ -1,23 +1,27 @@
 package cromwell.engine.backend.jes
 
 import cromwell.engine.backend.jes.authentication.JesDockerCredentials
-import cromwell.engine.io.gcs.GoogleConfiguration
+import cromwell.gcsfilesystem.{GoogleConfiguration, GoogleConfigurationAdapter}
 import cromwell.util.DockerConfiguration
 
 trait JesConfiguration {
   def jesConf: JesAttributes
-  def googleConf: GoogleConfiguration
+  def genomicsConf: GoogleConfiguration
+  def gcsConf: GoogleConfiguration
   def dockerConf: Option[JesDockerCredentials]
 }
 
 object ProductionJesConfiguration {
   lazy val jesConf =  JesAttributes()
-  lazy val googleConf =  GoogleConfiguration.gcloudConf.get
+  private lazy val oldGoogleConf =  GoogleConfigurationAdapter.gcloudConf.get
```

Comment:
Yeah I didn't want to touch too much on the configuration parsing since @mcovarr is going to shuffle everything anyway. This is all temporary and when we have the final conf, `GoogleConfigurationAdapter` will disappear.
I can try to think of a better name though.